src/main/java/io/aiven/kafka/connect/common/output/avro/AvroConfig.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.output.avro;
+
+import java.util.Map;
+
+import org.apache.avro.file.CodecFactory;
+
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+
+import io.aiven.kafka.connect.common.config.AivenCommonConfig;
+
+final class AvroConfig extends AbstractConfig {
+
+    AvroConfig(final Map<?, ?> originals) {
+        super(new ConfigDef(), originals);
+    }
+
+    CodecFactory codecFactory() {
+        final String codecName = originals()
+            .getOrDefault(AivenCommonConfig.FILE_COMPRESSION_TYPE_CONFIG, "null")
```

Comment:
This key is `file.compression.type`. This needs to be changed to something that is Avro specific as this is not the file compression but the codec that Avro uses to compress data in the Container File.

Reply:
Added `avro.codec` for Avro block compression. Default is `null` codec.